cmds/core/io/cmos.go

```diff
@@ -13,12 +13,16 @@ import (
 	"github.com/u-root/u-root/pkg/memio"
 )
 
-func init() {
-	usageMsg += `io (cr index)... # read from CMOS register index [14-127]
+const (
+	usageCmos = `io (cr index)... # read from CMOS register index [14-127]
```

Comment:
why usageCmos? We're in that command and file. What's wrong with just
usage

Reply:
I thought it would be more verbose to name the variable according to the file it originates from. I think that it looks cleaner when separated in the `const` section.

Reply:
This is an odd command, usage message depends on GOOS and GOARCH, but I actually agree for more succinct naming. See https://go.dev/wiki/CodeReviewComments#variable-names and if you still want to use verbose name it should be usageCMOS.